kolena/_experimental/object_detection/workflow.py

```diff
@@ -50,35 +52,38 @@ class TestSample(Image):
 class GroundTruth(BaseGroundTruth):
     """Ground truth type for the pre-built 2D Object Detection workflow."""
 
-    bboxes: List[LabeledBoundingBox]
+    objects: List[Union[LabeledBoundingBox, LabeledPolygon]]
     """
-    The ground truth [`LabeledBoundingBox`][kolena.workflow.annotation.LabeledBoundingBox]es associated with an image.
+    The ground truth [`LabeledBoundingBox`][kolena.workflow.annotation.LabeledBoundingBox]es or
+    [`LabeledPolygon`][kolena.workflow.annotation.LabeledPolygon]s associated with an image.
     """
 
-    ignored_bboxes: List[LabeledBoundingBox] = dataclasses.field(default_factory=list)
+    ignored_objects: List[Union[LabeledBoundingBox, LabeledPolygon]] = dataclasses.field(default_factory=list)
     """
-    The ground truth [`LabeledBoundingBox`][kolena.workflow.annotation.LabeledBoundingBox]es to be ignored in evaluation
-    associated with an image.
+    The ground truth [`LabeledBoundingBox`][kolena.workflow.annotation.LabeledBoundingBox]es or
+    [`LabeledPolygon`][kolena.workflow.annotation.LabeledPolygon]s to be ignored in evaluation associated with an
+    image.
     """
 
-    n_bboxes: int = dataclasses.field(default_factory=lambda: 0)
+    n_objects: int = dataclasses.field(default_factory=lambda: 0)
 
     def __post_init__(self):
-        object.__setattr__(self, "n_bboxes", len(self.bboxes))
+        object.__setattr__(self, "n_objects", len(self.objects))
 
 
 @dataclass(frozen=True)
 class Inference(BaseInference):
     """Inference type for the pre-built 2D Object Detection workflow."""
 
-    bboxes: List[ScoredLabeledBoundingBox]
+    objects: List[Union[ScoredLabeledBoundingBox, ScoredLabeledPolygon]]
```

Comment:
Would there be a list of mixed bboxes and polygons?

Reply:
Not commonly seen but they can coexist, neither has more significance than the other.